packages/app/src/containers/Canvas.tsx

```diff
@@ -249,6 +255,68 @@ export default memo(function Canvas() {
           event.preventDefault();
           break;
         }
+        case 'editPath': {
+          const layer: any = Selectors.getLayerAtPoint(
+            CanvasKit,
+            state,
+            insets,
+            rawPoint,
+            {
+              clickThroughGroups: false,
+              includeHiddenLayers: false,
+              includeArtboardLayers: false,
+            },
+          );
+          const boundingRects = getBoundingRectMap(
+            getCurrentPage(state),
+            state.selectedObjects,
+            {
+              clickThroughGroups: true,
+              includeArtboardLayers: false,
+              includeHiddenLayers: false,
+            },
+          );
+
+          if (!layer) return;
+
+          const boundingRect = boundingRects[layer.do_objectID];
+          //Why dose TS not recognize layer.points?
+          let selectedPoint: SelectedPoint | undefined = undefined;
+          layer.points.forEach((layer_point: any, index: number) => {
```

Comment:
Not all layers have points. First we need to filter down the selected layers. Maybe try:

```ts
getSelectedLayers(state).filter(Layers.isPointsLayer).forEach((layer) => { ... }
```

This way we have access to the `layer` and can avoid `getLayerAtPoint`